### PR TITLE
Bump version to 5.0.0-alpha1.snapshot1

### PR DESCRIPTION
**Pull request by suyograo**

### CODE REVIEW

File: logstash-core-event-java/lib/logstash-core-event-java/version.rb [hunk 1]
```diff
@@ -5,4 +5,4 @@
 # Note to authors: this should not include dashes because 'gem' barfs if
 # you include a dash in the version string.
 
-LOGSTASH_CORE_EVENT_JAVA_VERSION = "3.0.0.dev"
+LOGSTASH_CORE_EVENT_JAVA_VERSION = "5.0.0-alpha1.snapshot1"
```
Comment (ph) [inline logstash-core-event-java/lib/logstash-core-event-java/version.rb]: the packaging name and the version name are different I remember now why there is a gsub below.

**Comment by ph** [comment]
LGTM

File: logstash-core-event-java/lib/logstash-core-event-java/version.rb (hunk 1 above)
Comment (suyograo) [inline logstash-core-event-java/lib/logstash-core-event-java/version.rb]: yeah, remember https://github.com/elastic/logstash/blob/1.5/logstash-core.gemspec#L18

File: logstash-core-plugin-api/logstash-core-plugin-api.gemspec [hunk 2]
```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |gem|
   gem.require_paths = ["lib"]
   gem.version       = LOGSTASH_CORE_PLUGIN_API
 
-  gem.add_runtime_dependency "logstash-core", ">= 2.0.0", "<= 3.0.0.dev"
+  gem.add_runtime_dependency "logstash-core", "5.0.0.alpha1.snapshot1"
```
Comment (ph) [inline logstash-core-plugin-api/logstash-core-plugin-api.gemspec]: Correct pinning!

**Comment by ph** [comment]
LGTM, test fails is because we didnt not release the new version of the core yet.

```
Could not find gem 'logstash-core (= 5.0.0.alpha1.snapshot1) java' in any of the gem sources listed in your Gemfile or installed on this machine.
/home/jenkins/workspace/Logstash_PR/vendor/bundle/jruby/1.9/gems/bundler-1.9.10/lib/bundler/resolver.rb:328:in `verify_gemfile_dependencies_are_found!'
org/jruby/RubyArray.java:1613:in `each'

**Comment by purbon** [comment]
would be nice to have also https://github.com/elastic/logstash/pull/4911 in
the release, what do you think?

**Comment by ph** [comment]
@purbon yes

**Comment by ph** [comment]
LGTM

**Pull request merged**